src/Field/WfsSearch/WfsSearch.jsx

```diff
@@ -0,0 +1,389 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Select,
+  Spin
+} from 'antd';
+const Option = Select.Option;
+
+import Logger from '../../Util/Logger';
+
+import OlMap from 'ol/map';
+import OlFormatFilter from 'ol/format/filter';
+import OlFormatGeoJSON from 'ol/format/geojson';
+import OlFormatWFS from 'ol/format/wfs';
+
+/**
+ * The WfsSearch field.
+ * Implements an input field to do a WFS-GetFeature request.
+ *
+ * The GetFeature request is created with `ol.format.WFS.writeGetFeature`
+ * so most of the WFS specific options work like document in the corresponding
+ * API-docs: http://openlayers.org/en/latest/apidoc/ol.format.WFS.html#writeGetFeature
+ *
+ * @class WfsSearch
+ * @extends React.Component
+ */
+export class WfsSearch extends React.Component {
+
+  /**
+   * The className added to this component.
+   * @type {String}
+   * @private
+   */
+  className = 'react-geo-wfssearch'
+
+  static propTypes = {
+    /**
+     * A optional className that will be added to the input field.
+     */
+    className: PropTypes.string,
+    /**
+     * The base URL.
+     * @type {String}
+     */
+    baseUrl: PropTypes.string.isRequired,
+    /**
+     * The Property which should be shown in the inputfield when a selection was
+     * made.
+     * @type {String}
+     */
+    displayField: PropTypes.string,
```

Comment:
Can we change this to be a template or a function, so I can show multiple fields and arbitrary text? Also it would be super nice if we could somehow mark the matched textpart in the property.

Reply:
After having a look at the code i found out, that this prop is never used. I'll remove it. All what you expect should be realized in the `renderOption` prop.